### PR TITLE
Fix CI job - error status not reported

### PR DESCRIPTION
**Pull request by osinstom**

### CODE REVIEW

File: .github/workflows/test-integration.yml [hunk 1]
```diff
@@ -18,4 +18,4 @@ jobs:
           go-version: '1.13'
       - name: Run integration tests for PFCP Agent & UP4
         run: |
-            make test-up4-integration || docker-compose -f test/integration/infra/docker-compose.yml logs
+            make test-up4-integration || docker-compose -f test/integration/infra/docker-compose.yml logs; exit 1
```
Comment (pudelkoM) [inline .github/workflows/test-integration.yml:21]: Depending on how GHA structure the output (e.g. auto collapsed), could be always print the logs?

Comment (pudelkoM) [reply .github/workflows/test-integration.yml:21]: oh, there is a lot of output...

Maybe we should omit the `-v` on `go test`?
https://github.com/golang/go/issues/21461#issuecomment-348374826

Comment (osinstom) [reply .github/workflows/test-integration.yml]: it wouldn't print test case names then.

Comment (osinstom) [reply .github/workflows/test-integration.yml]: I believe that the current proposal makes sense

**Pull request merged**